.github/workflows/tf-dflook-apply.yaml

```
@@ -67,6 +67,9 @@ jobs:
 
       - name: Terraform Apply
         uses: dflook/terraform-apply@v1
+        env:
+          TERRAFORM_PRE_RUN:
+            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip -qq awscliv2.zip && ./aws/install
```

@swibrow Can we pin the version of the AWS CLI? 
```suggestion
            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.15.36.zip" -o "awscliv2.zip" && unzip -qq awscliv2.zip && ./aws/install
```

(or set it as a separate, easy to change, variable)

---

Added some more custom logic and set the aws cli version in an env so renovate can easily find and update.